### ISSUE TITLE
Suggestion: Clarifying Plugin Loading

### ISSUE THREAD

**Issue by han-solo318** [MiragianCycle/OVIWrite#16]
I had a bit of difficulty understanding during installation that not all plugins are enabled by default at startup. To enable them, you need to either use the `Lazy load all` command (or the `<leader>l` hotkey) or manually adjust the plugin configs to disable lazy loading. I believe others share this confusion (see https://github.com/MiragianCycle/OVIWrite/issues/8). While the hotkey is listed in WhichKey, it isn’t loaded by default, so new users may not realize this. It might be helpful to clarify in the documentation—perhaps in the readme or installation instructions—that plugins need to be manually loaded. Alternatively, enabling WhichKey by default could be considered, though I understand the potential concern for performance.

I can make a pull request to do any of those, but I wanted feedback before I just choose one when it might not be what you want.

Thank you for creating this great tool, I'm really happy to see an IWE in neovim!

**Comment by MiragianCycle** [comment]
Yes please.  Go ahead. I am sorry for the late response---been busy with actual writing. Send the PR with the proposed change and I will be happy to approve it. Thank you again.

**Comment by MiragianCycle** [comment]
I see you already done that. Hope this solves the issue?

**Comment by Valdus92** [comment]
How do you do the command `lazy load all`?

**Comment by sbishop333** [comment]
1. Open a CmdLine (Type a colon ":" in normal mode. No parentheses.)
   2. Lazy load all <RET> (Lazy must be capitalized)

Which two plugins do you want loaded? You could do Lazy load [plugin]
for each.

**Comment by Valdus92** [comment]
All preferably, but just the ones to make the program go smoothly.

**Comment by Valdus92** [comment]
That lazy cut out worked. Now two things. 

1. Can I get back to normal Neovim, ie shut down oviwrite without shutting neovim?
2. Do we have guide on basic commands and such?

**Comment by MiragianCycle** [comment]
> How do you do the command `lazy load all`?

I am late for this sorry. The key binding `<space> L A L` loads all plugins. I usually open OVIwrite, then <space> L A L, before opening the file. You might get errors, which I am yet to get to, but those errors won't affect your ability to write. If you want to turn off notifications then `:NoiceDisable` will remove the notifications for the session. 

Alternatively, you can add the following auto-command to `globals.lua' which ensure that noice is disabled depending on file-type. So something like 

```
`-- The following ensure that Noice notifications are disabled when I enter a buffer of either *.tex or *.wiki



vim.api.nvim_create_autocmd("BufEnter", {
  pattern = "*.wiki",
  callback = function()
    vim.cmd("NoiceDismiss")
  end,
})

vim.api.nvim_create_autocmd("BufEnter", {
  pattern = "*.tex",
  callback = function()
    vim.cmd("NoiceDismiss")
  end,
})`
```
